db/schema.go

```diff
@@ -82,6 +85,12 @@ func Prepare(conn *pgx.Conn, schemas []string, includeTables, excludeTables []st
 		log.Infof("registering trigger for table %s", table)
 		err = registerTrigger(tx, table)
 		if err != nil {
+			pgErr, ok := err.(pgx.PgError)
+			if ok {
+				log.Printf("%v+", pgErr)
+			}
+			// Failure case resolution:
+			// GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO master;
```

Comment:
This is RDS specific, please remove this.

Reply:
removed